defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+LOAD_BALANCER_SERVER_IP: "{{ lookup('dig', ansible_host) }}"
```

Comment:
We'll need to add `dnspython` to `requirements.txt` in deployment-secrets-services. (Maybe this has already been addressed)